src/components/TopBar/OhmMenu.jsx

```diff
@@ -18,7 +18,9 @@ import SOhmImg from "src/assets/tokens/token_sOHM.svg";
 import WsOhmImg from "src/assets/tokens/token_wsOHM.svg";
 import token33tImg from "src/assets/tokens/token_33T.svg";
 
-const addTokenToWallet = (tokenSymbol, tokenAddress) => async () => {
+import { segmentUA } from "../../helpers/userAnalyticHelpers";
+
+const addTokenToWallet = (tokenSymbol, tokenAddress, address) => async () => {
```

Comment:
you could prolly pull this down into `OhmMenu` then you wouldn't have to pass `address` as a param, but... it works 👍